packages/wrangler/src/parse.ts

```diff
@@ -76,7 +76,7 @@ type TomlError = Error & {
 export function parseTOML(input: string, file?: string): TOML.JsonMap | never {
   try {
     // Normalize CRLF to LF to avoid hitting https://github.com/iarna/iarna-toml/issues/33.
-    const normalizedInput = input.replace(/\r\n$/g, "\n");
```

Comment:
what did removing `$` from the RegEx do?

Reply:
Here `$` means "end of the string". So it was only replacing the very last `\r\n`.